table.go

```diff
@@ -213,11 +213,11 @@ func (rt *RoutingTable) addPeer(p peer.ID, queryPeer bool, isReplaceable bool) (
 
 	// the bucket to which the peer belongs is full. Let's try to find a peer
 	// in that bucket which is replaceable.
-	replaceablePeer := bucket.findFirst(func(p *PeerInfo) bool {
-		return p.replaceable
+	replaceablePeer := bucket.min(func(p1 *PeerInfo, p2 *PeerInfo) bool {
+		return p1.replaceable
```

Comment:
maybe add a comment here about us not needing a stable sort here just to call it out

Reply:
Done.